### PR TITLE
[hi] Localize 'API Gateway' to Hindi

### PR DESCRIPTION
**Pull request by AniketNS**
### Checklist before opening this PR (put `x` in the checkboxes)
- [ ] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [ ] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Review by kumarankit999 — COMMENTED** [review]
lgtm, What others says??

File: content/hi/api-gateway.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+status: Completed
+category: अवधारणा
+tags: ["networking", "", ""]
+---
+
+## यह क्या है
+
+[एपीआई](/application-programming-interface/) गेटवे एक ऐसा उपकरण है जो अद्वितीय एप्लिकेशन एपीआई को एकत्र करता है, जिससे वे सभी एक ही स्थान पर उपलब्ध हो जाते हैं।
+यह संगठनों को प्रमुख कार्यों को स्थानांतरित करने की अनुमति देता है, जैसे प्रमाणीकरण और प्राधिकरण या अनुप्रयोगों के बीच अनुरोधों की संख्या को केंद्रीय रूप से प्रबंधित स्थान पर सीमित करना। 
+एपीआई गेटवे (अक्सर बाहरी) एपीआई उपभोक्ताओं के लिए एक सामान्य इंटरफ़ेस के रूप में कार्य करता है।
```
Comment (abhay-raj19) [inline content/hi/api-gateway.md:12]: एपीआई गेटवे अक्सर बाहरी उपभोक्ताओं के लिए एक सामान्य इंटरफ़ेस के रूप में कार्य करता है।

Comment (abhay-raj19) [inline content/hi/api-gateway.md:12]: एपीआई गेटवे अक्सर बाहरी उपभोक्ताओं के लिए एक सामान्य इंटरफ़ेस के रूप में कार्य करता है।

**Review by abhay-raj19 — CHANGES_REQUESTED** [review]
And else looks good to me .

**Comment by AniketNS** [comment]
I've made the changes you told @abhay-raj19. Will you please check and tell me if is there anything I should do?

**Comment by jihoon-seo** [comment]
@AniketNS Please change the target branch of this PR to `dev-hi`, not `main`.

**Review by bishal7679 — COMMENTED** [review]
Left some suggestions!

**Comment by bishal7679** [comment]
> @AniketNS Please change the target branch of this PR to `dev-hi`, not `main`.

+1

**Comment by bishal7679** [comment]
@AniketNS Its always a good practise to create your local branch significant
you should signoff your commit as well to avoid DCO failure! use `git commit -s -m "your message here"` in this way

**Review by aj11anuj — COMMENTED** [review]
@AniketNS, Why you opened a duplicate PR here, when you were already working on #1614.

**Review by kumarankit999 — COMMENTED** [review]
Yes @aj11anuj, Why are you creating Duplicate PR for this Since #1614 is opened

**Comment by AniketNS** [comment]
Hey @aj11anuj and @kumarankit999, I don't How this new PR got created. As I'm a beginner, I don't actually how it is happening. Really sorry for that.

**Comment by kumarankit999** [comment]
No Problem @AniketNS, you can work on that PR Only!

**Comment by aj11anuj** [comment]
Approvers, please check for duplication before starting review process.

And contributors are also advised to check if there's already an open PR or not, before making a new PR

Closing this one as it's a duplicate

**Pull request closed without merging**